app/packages/state/src/hooks/useExpandSample.ts

```diff
@@ -34,9 +34,9 @@ export default <T extends Lookers>(store: LookerStore<T>) => {
             schemaAtoms.activeFields({ modal: false }),
           ],
           [atoms.cropToContent(true), atoms.cropToContent(false)],
-          [atoms.colorSeed(true), atoms.colorSeed(false)],
+          [atoms.colorSeed, atoms.colorSeed],
```

Comment:
`[atoms.colorSeed, atoms.colorSeed]` and `[atoms.alpha, atoms.alpha]` can be removed. They have no effect